### PR TITLE
 Add support for list of possible values

### PR DESCRIPTION
**Pull request by nitriques**
This allows us to set a comma separated list of values which as valid
for the state.

### CODE REVIEW

File: src/modules/auto-sync-state-from-qs.js [hunk 1]
```diff
@@ -73,6 +81,9 @@
 		return {
 			page: {
 				fragmentChanged: onFragmentChanged
+			},
+			articleChanger: {
+				enter: onFragmentChanged
```
Comment (fhamon) [inline src/modules/auto-sync-state-from-qs.js]: `onFragmentChanged`
https://shft.cl/pc1iwg7x2

Comment (fhamon) [reply src/modules/auto-sync-state-from-qs.js]: I know it does the same thing, but my opinion is that we should put it in it's own method with an appropriate name like `onArticleEnter` or something 😁

Comment (nitriques) [reply src/modules/auto-sync-state-from-qs.js]: @fhamon check dd4a1ac

**Comment by nitriques** [comment]
@DeuxHuitHuit/review ping !

**Comment by nitriques** [comment]
@PascalPiche still not convinced ?

**Comment by nitriques** [comment]
@DeuxHuitHuit/review guys ?

**Comment by nitriques** [comment]
@DeuxHuitHuit/review guys ;)

**Comment by nitriques** [comment]
I'll merge if I do not get any feedback @fhamon @PascalPiche

**Comment by PascalPiche** [comment]
Plz no, Im sure this will break some base behavior actually working. I currently setting up the test framework and will try to make test for this module as soon as possible.

**Pull request merged**